frontend/src/components/regions/UserDashboard/utils.js

```diff
@@ -0,0 +1,32 @@
+import { cardColors } from "../../../colors";
+
+/**
+ * fillColor function returns a colour string value based on the type of column:
+ * Grey for Blocked column,
+ * Blue for Ready column,
+ * Green for In Progress column,
+ * Red for Review Feedback column,
+ * Orange for Review column,
+ * Yellow for Complete column
+ */
+
+export const fillColor = (columnName) => {
```

Comment:
This seems a bit overkill. There is a much simpler way. If we just treat data as data then you dont even need a function to work this stuff out.